app/Http/Controllers/Maps/CustomMapDataController.php

```diff
@@ -198,6 +198,11 @@ public function get(Request $request, CustomMap $map): JsonResponse
                 $nodes[$nodeid]['device_image'] = $node->device->icon;
                 $nodes[$nodeid]['device_info'] = Blade::render('<x-device-link-map :device="$device" />', ['device' => $node->device]);
 
+                $warning_time = \LibreNMS\Config::get('uptime_warning', 86400);
```

Comment:
You can use `Config::get` rather than `\LibreNMS\Config::get` on this line and below

Reply:
Thanks. Stole it from nearby code, that'll teach me!  Fixed now.